packages/react-storage/src/components/StorageBrowser/Views/Controls/__tests__/Download.spec.tsx

```diff
@@ -63,22 +56,30 @@ describe('DownloadControl', () => {
     expect(icon).toHaveAttribute('aria-hidden', 'true');
   });
 
-  it('calls downloadData onClick', async () => {
-    // @ts-expect-error
-    downloadSpy.mockResolvedValueOnce({ key: 'a_key' });
+  it('calls getUrl onClick', async () => {
```

Comment:
This test runs the `download` action too, but I couldn't get the mocks to correctly work for `document.location` when the `click()` happens and so there is a console error that gets printed during the tests